input/pages/operations.md

```diff
@@ -92,11 +92,49 @@ Options for federated client specification:
     * contentEndpoint
     * terminologyEndpoint
 * Resource/$draft
+    * id
+    * url
+    * version
+    * identifier
+    * resource
 * Resource/$approve
+    * id
+    * url
+    * version
+    * identifier
+    * resource
+    * approvalDate
+    * artifactAssessmentType
+    * artifactAssessmentSummary
+    * artifactAssessmentTarget
+    * artifactAssessmentRelatedArtifact
+    * artifactAssessmentAuthor
 * Resource/$revise
+    * id
+    * url
+    * version
+    * identifier
+    * resource
 * Resource/$review
+    * id
+    * url
+    * version
+    * identifier
+    * resource
 * Resource/$endorse
+    * id
+    * url
+    * version
+    * identifier
+    * resource
```

Comment:
The list of operations in `operations.md` isn't the same as the ones in `artifact-repository-service.md`, is that intentional or does it need to be changed?

Reply:
Agreed.